hcl2/api.py

```diff
@@ -15,4 +15,4 @@ def loads(text: str) -> dict:
     # Lark doesn't support a EOF token so our grammar can't look for "new line or end of file"
     # This means that all blocks must end in a new line even if the file ends
     # Append a new line as a temporary fix
-    return hcl2.parse(text + "\n")
+    return hcl2.parse(text + "\n")  # type: ignore
```

Comment:
`mypy` was complaining that `parse` returns a `Tree` instead of a `Dict`. We can leave the ignore here but was curious why it does that.